### ISSUE TITLE
Faulty core fixes: the case of the moving space

### ISSUE THREAD

**Issue by jrfnl** [WordPress-Coding-Standards/WordPress-Coding-Standards#977]
#### Summary:
While https://github.com/squizlabs/PHP_CodeSniffer/pull/1404 definitely is a huge improvement in the spaces -> tabs fixing, I'm still seeing an issue with this sniff when there is a combination of tabs and spaces for indentation and the spaces are not there for alignment.

Most likely cause: `Generic.WhiteSpace.DisallowSpaceIndent`

#### Example:

File: `./src/wp-admin/update.php` line 127 (and in lots of other places too)

Source:
```php
	 		wp_die( $api ); // Tab - Space - Tab - Tab
```

Fixed as:
```php
			 wp_die( $api ); // Tab - Tab - Tab - Space
```

Expected fix:
```php
			wp_die( $api ); // Tab - Tab - Tab
```

**Comment by jrfnl** [comment]
A fix for this has now been pulled upstream: https://github.com/squizlabs/PHP_CodeSniffer/pull/1599
The PR also addresses a secondary issue, i.e. that subsequent lines of a multi-line non-doc-block comment were not fixed by the upstream sniff.

### Analysis of this bug and the effect of the upstream fix

To analyse the effect, I've used a combination of running the new `WordPress.WhiteSpace.PrecisionAlignment` sniff - PR #1099 - and both the old and the improved version of the upstream `Generic.WhiteSpace.DisallowSpaceIndent` sniff against the WP Core `/src` directory using the `phpcs.xml.5.dist` ruleset.

`Generic.WhiteSpace.DisallowSpaceIndent`: | Old (upstream `master`) | Improved (upstream PR)
------------------------------------------- | ------------------------ | --------------------------
Original precision alignment issues found | 399 | 399
Nr of issues fixed by the upstream sniff | 423 in 90 files | 440 in 92 files (**+17**)
Precision alignment issues found after fixing by upstream sniff | 539 (**+140**) | 310 (**-89**)

**Comment by jrfnl** [comment]
Wowie! Just in time for the core patch, the upstream PR for this issue has just been merged ;-) /cc @pento
